src/ifcgeom/IfcIShapeProfileDef.cpp

```diff
@@ -65,8 +93,25 @@ bool IfcGeom::Kernel::convert(const IfcSchema::IfcIShapeProfileDef* l, TopoDS_Sh
 		IfcGeom::Kernel::convert(l->Position(), trsf2d);
 	}
 
-	double coords[24] = {-x1,-y, x1,-y, x1,-y+dy1, d1,-y+dy1, d1,y-dy2, x2,y-dy2, x2,y, -x2,y, -x2,y-dy2, -d1,y-dy2, -d1,-y+dy1, -x1,-y+dy1};
-	int fillets[4] = {3,4,9,10};
-	double radii[4] = {f1,f2,f2,f1};
-	return util::profile_helper(12,coords,(doFillet1||doFillet2) ? 4 : 0,fillets,radii,trsf2d,face);
+	double coords[24] = {
```

Comment:
I had to quintuple check. I made the following diagram.

![afbeelding](https://user-images.githubusercontent.com/1096535/211195355-54da9dca-8161-48cc-b82d-b2724aa7796b.png)

But I think `-y` should always be accompanied with `+ft1` and `y` should always be accompanied with `-ft2`. `-y` is the bottom flange and should always be paired with bottom flange thickness `ft1`.

Reply:
Are you sure? `-y` is half the overall depth, and so e.g. the first coordinate `-x1,-y,` should represent the bottom left coordinate, I don't see why there should be `+ft1`?

![2023-01-09-083711_833x676_scrot](https://user-images.githubusercontent.com/88302/211220417-ab6d76ba-6209-4069-b751-77bb0b70d1a9.png)

Reply:
Ok, sorry, "always accompanied" unnecessarily confusing, what I meant, but didn't clearly articulate -y likely should never incur a +ft2 only +ft1.

```
       vvv
 x1,-y+ft1-dy2,
 d1,-y+ft1+dy1,
...
-d1,-y+ft2+dy1,
-x1,-y+ft2-dy2
       ^^^
```